### PR TITLE
Feat/freze e2e tests

### PR DESCRIPTION
**Pull request by calvogenerico**

### CODE REVIEW

File: packages/e2e/.gitignore
Comment (aon) [inline packages/e2e/.gitignore:1]: This file is unnecessary

File: packages/e2e/suites/full/freeze.spec.ts [hunk 1]
```diff
@@ -0,0 +1,536 @@
...
+import type { Page } from "@playwright/test";
+import type { Dappwright as Wallet } from "@tenkeylabs/dappwright";
+import type { COUNCIL_SIZE } from "@repo/contracts/helpers/constants";
+import { TestApp } from "./helpers/test-app.js";
+
+const testApp = new TestApp();
+
+test.beforeEach(async ({ page }) => {
+  await testApp.reset();
+  await page.goto("/");
```
Comment (aon) [inline packages/e2e/suites/full/freeze.spec.ts:11]: Test automatically goes to "/" before each test.

File: packages/e2e/suites/full/freeze.spec.ts [hunk 2]
```diff
@@ -0,0 +1,536 @@
...
+  await expect(page.getByText("Pending proposal already exists.")).toBeVisible();
+});
+
+test("TC313: Attempt to approve soft freeze by guardian, sign button is not displayed", async ({
+  page,
+  switcher,
+}) => {
+  await switcher.guardian(1, page);
+  await createFreeze(page, "soft");
+  await assertCannotSignProposal(page, "soft");
```
Comment (aon) [inline packages/e2e/suites/full/freeze.spec.ts:193]: Doesn't this assert fail? As it tries to create a freeze by itself when a soft freeze already exists.

File: packages/e2e/suites/full/freeze.spec.ts [hunk 3]
```diff
@@ -0,0 +1,536 @@
...
+// change threshold
+
+async function cannotSignChangeThreshold(page: Page) {
+  await createChangeThreshold(page);
+
+  await goToFreezeDetailsPage(page, "change-threshold");
+  await expect(page.getByText("No role actions")).toBeVisible();
+  const approveButtons = await page.getByRole("button", { name: "Approve" }).all();
+  expect(approveButtons).toHaveLength(0);
+}
```
Comment (aon) [inline packages/e2e/suites/full/freeze.spec.ts:419]: Can't we reuse the `assertCannotSignProposal` function here?

File: packages/e2e/.gitignore
Comment (calvogenerico) [reply packages/e2e/.gitignore:1]: Playwright generats that thing, and makes the linter to die.

File: apps/web/app/components/basic-sign-button.tsx [hunk 4]
```diff
@@ -33,6 +33,7 @@ type SignButtonProps = {
   types: TypedMessgeTypes;
   message: Record<string, string | number | bigint>;
   primaryType: string;
+  testId: string;
```
[the remark this answers is not in the document]
Comment (calvogenerico) [reply apps/web/app/components/basic-sign-button.tsx]: This is a prop name, so it cannot have hiphens. I renamed to "dataTestId"

File: packages/e2e/suites/full/freeze.spec.ts (hunk 3 above)
Comment (calvogenerico) [reply packages/e2e/suites/full/freeze.spec.ts:418]: Well... Maybe. The issue is that those are slightly different in an awful way. I wanted to have linear code for this tests. Avoid conditionals as much as possible.

If I accept adding conditionals I can remove the duplicated functions for change threshold. Let me try it to see how it looks.

**Review by aon — APPROVED** [review]

**Pull request merged**